.github/workflows/update_fr_template.yaml

```diff
@@ -0,0 +1,52 @@
+name: Update repo from template
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
Should this start out as a cron?
I would vote to make this through a PR that is tagged as 'TEMPLATE-UPDATE' or something.

Reply:
I left it as a cron but it will commit specifically to the `template-updates` branch (which we'll have to create manually for now). I'd be more than happy for it to create a tagged PR but I'm guessing that will take me more than the 10 minutes I have this afternoon to implement!